### DISCUSSION TITLE
Panel Inside Panel

### DISCUSSION THREAD

**Discussion by lunabunn** [emilk/egui#517] [Q&A]
Hello. I want to create resizable vertical partitions (like TopBottomPanel) inside a SidePanel. Is there a nice way to do this?

**Comment by parasyte** [comment]
As far as I am aware, this isn't easily doable right now, but you could probably create these kinds of panels yourself. See this other discussion for more info: https://github.com/emilk/egui/discussions/469#discussioncomment-836880
